docs/install.md

```diff
@@ -0,0 +1,204 @@
+# Installing Tekton Dashboard
+
+This guide explains how to install Tekton Dashboard. It covers the following topics:
+
+* [Before you begin](#before-you-begin)
+* [Pre-requisites](#pre-requisites)
+* [Which version should I use](#which-version-should-i-use)
+* [Installing Tekton Dashboard on Kubernetes](#installing-tekton-dashboard-on-kubernetes)
+* [Installing Tekton Dashboard on OpenShift](#installing-tekton-dashboard-on-openshift)
+* [Accessing the Dashboard on Kubernetes](#accessing-the-dashboard-on-kubernetes)
+* [Accessing the Dashboard on OpenShift](#accessing-the-dashboard-on-openshift)
+* [Uninstalling the Dashboard on Kubernetes](#uninstalling-the-dashboard-on-kubernetes)
+* [Next steps](#next-steps)
+
+## Before you begin
+
+1. Choose the version of Tekton Dashboard you want to install. You have the following options:
+
+   * **[Official](https://github.com/tektoncd/dashboard/releases)** - install this unless you have a specific reason to go for a different release.
+   * **[Nightly](../tekton/README.md#nightly-releases)** - may contain bugs, install at your own risk. Nightlies live at `gcr.io/tekton-nightly`.
+   * **[`HEAD`]** - this is the bleeding edge. It contains unreleased code that may result
+     in unpredictable behavior. To get started, see the [development guide](../DEVELOPMENT.md) instead of this page.
+
+2. If you don't have an existing Kubernetes cluster, set one up, version 1.15 or later:
+
+   ```bash
+   #Example command for creating a cluster on GKE
+   gcloud container clusters create $CLUSTER_NAME \
+     --zone=$CLUSTER_ZONE --cluster-version=1.15.11-gke.5
+   ```
+
+3. Grant `cluster-admin` permissions to the current user:
+
+   ```bash
+   kubectl create clusterrolebinding cluster-admin-binding \
+   --clusterrole=cluster-admin \
+   --user=$(gcloud config get-value core/account)
+   ```
+
+   See [Role-based access control](https://cloud.google.com/kubernetes-engine/docs/how-to/role-based-access-control#prerequisites_for_using_role-based_access_control)
+   for more information.
+
+## Pre-requisites
+
+In order to install the Tekton Dashboard, please make sure the following requirements are met:
+- A [kubernetes cluster](https://github.com/kubernetes/kubernetes) must be installed with version 1.15.0 or later.
```

Comment:
already mentioned above, should that be moved to this section or removed?

Reply:
Kept as i removed the mention above